frontend/src/patched/JIT.js

```diff
@@ -8463,26 +8519,24 @@ Layouts.Radial = new Class({
    - <Options.Tips>
    - <Options.NodeStyles>
    - <Options.Navigation>
-   
+
    Additionally, there are other parameters and some default values changed
-   
+
    interpolation - (string) Default's *linear*. Describes the way nodes are interpolated. Possible values are 'linear' and 'polar'.
-   levelDistance - (number) Default's *100*. The distance between levels of the tree. 
-     
+   levelDistance - (number) Default's *100*. The distance between levels of the tree.
+
    Instance Properties:
 
    canvas - Access a <Canvas> instance.
    graph - Access a <Graph> instance.
    op - Access a <RGraph.Op> instance.
    fx - Access a <RGraph.Plot> instance.
-   labels - Access a <RGraph.Label> interface implementation.   
+   labels - Access a <RGraph.Label> interface implementation.
 */
 
 $jit.RGraph = new Class( {
 
-  Implements: [
-      Loader, Extras, Layouts.Radial
-  ],
+  Implements: [ Loader, Extras, Layouts.Radial ],
```

Comment:
was this some auto styler? I was thinking this could be reverted to be on 3 lines